.azuredevops/modulePipelines/ms.web.hostingenvironments.yml

```diff
@@ -51,6 +51,7 @@ stages:
           removeDeployment: '${{ parameters.removeDeployment }}'
           deploymentBlocks:
             - path: $(modulePath)/.parameters/parameters.json
+          defaultJobTimeoutInMinutes: 360
```

Comment:
```suggestion
          defaultJobTimeoutInMinutes: 180
```

Even our longest pending deployments took 'only' 2:40 hours. We should try to set a conservative limit to not waste time in case the pipeline just stalls